### PR TITLE
Wtc 37 vertical slider

### LINKED ISSUE #37: Change banana slider to be vertical

### PR DESCRIPTION
**Pull request by skibinska**
fixes #37 

- remove checkboxes
- change banana slider to be vertical
- add banana tree to banana slider

### CODE REVIEW

File: slider.js [hunk 1]
```diff
@@ -2,12 +2,12 @@
...
 	var max = $(".slider__range").attr("max");
-	var horizontal;
+	var vertical;
 
 	window.updateOutput = function (figure) {
 		$(".slider__banana").html(figure);
-		horizontal = figure/max*($(".eating__slider").width()-$(".slider__reel").width()) + 'px';
-		$(".slider__banana, .slider__reel").css({left: horizontal});
+		vertical = figure/max*($(".eating__slider").height()-$(".slider__reel").height()) + 'px';
+		$(".slider__banana, .slider__reel").css({bottom: vertical});
```
Comment (des-des) [inline slider.js]: indentation.. Is this code copy-pasted? if from stack overflow should leave a link..

Comment (des-des) [inline slider.js]: sry I know not from this change set but missed this before.

**Comment by skibinska** [comment]
I added the link below the code and rewrite this file.

File: slider.js [hunk 2]
```diff
@@ -11,3 +11,5 @@
 	}
 	updateOutput($(".slider__range").val());
 })(jQuery);
+
+// http://thecodeplayer.com/walkthrough/spicing-up-the-html5-range-slider-input
```
Comment (des-des) [inline slider.js]: :tada:

**Pull request merged**